integration_tests/pages/page.ts

```diff
@@ -5,9 +5,18 @@ export default abstract class Page {
     return new constructor()
   }
 
-  protected constructor(private readonly pageId: string, private readonly axeTest = true) {
+  protected constructor(
+    private readonly pageId: string,
+    private readonly options: { axeTest?: boolean; expectHelpdeskLink?: boolean } = {
+      axeTest: true,
+      expectHelpdeskLink: true,
```

Comment:
We expect the helpdesk link to be on every page, but in the event we get a requirement that says "Oh yeah, but not _this_ page" then we can handle that 👍